src/components/BottomNav/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as S from "./bottomNav.styled";
+import { PATH } from "../../constants/path";
+import { useLocation } from "react-router-dom";
+
+const BottomNav = () => {
+  const location = useLocation();
```

Comment:
```suggestion
  const { pathname } = useLocation();
```
useLocation 훅에서 pathname만 사용하신다면 이렇게 해도 될 것 같습니다!

Reply:
넵 수정하였습니다. 그런데 여기서 Commit suggestion 버튼만 눌러도 수정이 되는거 같군요! 신기합니다ㅋㅋ